kernel/lib/semaphore.cpp

```diff
@@ -131,6 +132,8 @@ int px_sem_post(px_sem_t *sem) {
 }
 
 uint32_t px_sem_getval(px_sem_t *sem, uint32_t *val) {
+    // FIXME: I don't think val should be unused?
+    (void)val;
```

Comment:
Probably not. Can you make an issue for this?

Reply:
#151 is now open